tests/Functional/Fixtures/Symfony/AppKernel.php

```diff
@@ -4,13 +4,18 @@
 
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\HttpKernel;
 use Symfony\Component\HttpKernel\HttpKernelInterface;
 
-class AppKernel extends HttpKernel
+/**
+ * A simplistic kernel that is actually the whole application.
+ */
+class AppKernel implements HttpKernelInterface
```

Comment:
so actually extending the HttpKernel was only creating problems. the simplistic kernel just needs a handle method!